.codecov.yml

```
@@ -0,0 +1,39 @@
+  behavior: default
+  require_changes: no
+
+ignore:
+  - "go/cmd/gitbak/main.go"       # Main entry point
+  - "go/build/**/*"               # Build artifacts
+  - "go/test/**/*"                # Test files
+  - "sh/**/*"                     # Shell script implementation
+  - "**/*_test.go"                # Test files (coverage shown in code being tested)
+  - ".github/**/*"                # GitHub workflow files
+  - "docs/**/*"                   # Documentation
+  - "**/*.md"                     # Markdown files
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)